src/resolvers/ReviewResolver.ts

```diff
@@ -0,0 +1,277 @@
+                    dropDown.value = dropDownInput.value;
+                    dropDown.reviewQuestion = question;
+                    await dropDown.save();
+                }
+            }
+        }
+
+        const result = await ReviewCampaign.findOne({
+            where: { id: campaign.id },
+            relations: ["questions", "questions.dropDown"]
+        });
+
+        return { reviewCampaign: result };
+    }
+
+    @Mutation(() => ReviewCampaign)
+    @UseMiddleware(isAuth)
+    async updateReviewCampaign(@Arg("input") input: UpdateReviewCampaignInput) {
+        const campaign = await ReviewCampaign.findOne({ where: { id: input.id } });
+        if (!campaign) {
+            throw new Error("Review campaign not found");
+        }
+
+        if (input.title !== undefined) campaign.title = input.title;
+        if (input.description !== undefined) campaign.description = input.description;
+        if (input.startDate !== undefined) campaign.startDate = input.startDate ? new Date(input.startDate) : null;
+        if (input.endDate !== undefined) campaign.endDate = input.endDate ? new Date(input.endDate) : null;
+        if (input.for !== undefined) campaign.for = input.for;
+        if (input.for_id !== undefined) campaign.for_id = input.for_id;
+
+        await campaign.save();
+        return campaign;
+    }
+
+    @Mutation(() => Boolean)
+    @UseMiddleware(isAuth)
+    async deleteReviewCampaign(@Arg("id") id: string) {
+        const campaign = await ReviewCampaign.findOne({ where: { id } });
+        if (!campaign) {
+            throw new Error("Review campaign not found");
+        }
+
+        await campaign.remove();
+        return true;
+    }
+
+    // Question Mutations
+    @Mutation(() => ReviewQuestion)
+    @UseMiddleware(isAuth)
+    async addQuestionToCampaign(@Arg("input") input: AddQuestionToCampaignInput) {
+        const campaign = await ReviewCampaign.findOne({ where: { id: input.campaignId } });
+        if (!campaign) {
+            throw new Error("Review campaign not found");
+        }
+
+        const question = ReviewQuestion.create({
+            question: input.question.question,
+            questionType: input.question.questionType,
+            rangeMin: input.question.rangeMin,
+            rangeMax: input.question.rangeMax,
+            reviewCampaign: campaign
+        });
+
+        await question.save();
+
+        // Create dropdown options if needed
+        if (input.question.dropDown && input.question.questionType === "dropdown") {
+            for (const dropDownInput of input.question.dropDown) {
+                const dropDown = DropDown.create({
+                    text: dropDownInput.text,
+                    value: dropDownInput.value,
+                    reviewQuestion: question
+                });
+                await dropDown.save();
+            }
+        }
+
+        return question;
+    }
+
+    @Mutation(() => ReviewQuestion)
+    @UseMiddleware(isAuth)
```

Comment:
Before you can send review_campaign you need to verify user is logged in and authenticated. And your sending them the right review, and not sending them the wrong thing